tests/libres_tests/res/enkf/test_subst_config.py

```diff
@@ -17,23 +17,22 @@ def without_key(a_dict, key):
     return {index: value for index, value in a_dict.items() if index != key}
 
 
-@pytest.fixture
-def snake_oil_structure_config(copy_case):
+@pytest.fixture(name="snake_oil_structure_config")
+def fixture_snake_oil_structure_config(copy_case):
     copy_case("snake_oil_structure")
     return {
         ConfigKeys.RUNPATH_FILE: "runpath",
         ConfigKeys.CONFIG_DIRECTORY: os.getcwd(),
         ConfigKeys.CONFIG_FILE_KEY: "config",
         ConfigKeys.DEFINE_KEY: {"keyA": "valA", "keyB": "valB"},
         ConfigKeys.DATA_KW_KEY: {"keyC": "valC", "keyD": "valD"},
-        ConfigKeys.DATA_FILE: "eclipse/model/SNAKE_OIL.DATA",
```

Comment:
Is it really necessary to modify the test in this way?

Reply:
i think i didn't test if it works with it, or not - i think i removed it because subst config no longer uses data file from the config.
so you might call it misleading to keep it in there, or just non minimalistic.
if you want, i can put in a separate case that tests whether we get the expected behaviour when DATA_FILE is present - is this desired?

Reply:
It would be nice to know why the test fails without removing `ConfigKeys.DATA_FILE` and then add some testing for the new behavior.

Reply:
but but but... i don't think it's failing...
i can try to see what happens when i add it back

Reply:
as i didn't remember, but intuitively suspected ; ) the test doth not faileth when one adds the DATA FILE - it has no effect, as should be expected. so i think we can just leave it out. i'm gonna resolve this.

feel free to merge yourself, whenever you approve!